astroid/manager.py

```diff
@@ -52,6 +52,8 @@ def __init__(self):
             # Export these APIs for convenience
             self.register_transform = self._transform.register_transform
             self.unregister_transform = self._transform.unregister_transform
+            self.max_inferable = int(
```

Comment:
Let's default it to 100 by default here, if someone wants to control the value, it should set it against the `Manager` itself.